elpd_dashboard.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #146.    pn.Column(pn.Row(model_selection1,model_selection2),
</pre><p>Enclose this into <code>display()</code>and all the content of this notebook into a function. Then calling the function will generate the dashboard.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_dashboard/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>